templates/project/.gitattributes.twig

```diff
@@ -6,3 +6,7 @@
 *.md export-ignore
 {{ tests_path }} export-ignore
 {{ docs_path }} export-ignore
+Makefile export-ignore
+phpunit.xml.dist export-ignore
+phpstan-baseline.neon export-ignore
+phpstan.neon.dist export-ignore
```

Comment:
How do you feel about keeping these rules sorted?

I recommend the [Lines Sorter](https://plugins.jetbrains.com/plugin/5919-lines-sorter) plugin for PhpStorm.

Reply:
Open for a PR? :-)

Reply:
I've sorted those lines.